src/Amp.php

```diff
@@ -151,7 +151,9 @@ public static function isRuntimeScript(DOMNode $node)
 
         if (
             substr($src, -6) !== '/v0.js'
+            && substr($src, -7) !== '/v0.mjs'
             && substr($src, -14) !== '/amp4ads-v0.js'
+            && substr($src, -15) !== '/amp4ads-v0.mjs'
```

Comment:
The more comparisons we have here, the more likely it is that a single regex might be faster. Maybe we can just add a TODO here to check performance later on?
```suggestion
			// @TODO Compare performance against single regex.
            substr($src, -6) !== '/v0.js'
            && substr($src, -7) !== '/v0.mjs'
            && substr($src, -14) !== '/amp4ads-v0.js'
            && substr($src, -15) !== '/amp4ads-v0.mjs'
```